### PR TITLE
docs: revert-multiRM-mc-doc

### PR DESCRIPTION
**Pull request by tara-det-ai**
to cherry pick for release

revert #9050 #9122

### CODE REVIEW

File: docs/reference/deploy/master-config-reference.rst [hunk 1]
```diff
@@ -241,30 +241,6 @@ otherwise active (as defined by the ``notebook_idle_type`` option in the :ref:`t
 
 The resource manager used to acquire resources. Defaults to ``agent``.
```
Comment (NicholasBlaskey) [inline docs/reference/deploy/master-config-reference.rst:243]: I don't think this part got reverted?
https://github.com/determined-ai/determined/pull/9122

Comment (tara-det-ai) [reply docs/reference/deploy/master-config-reference.rst:243]: you are correct; adding that

Comment (NicholasBlaskey) [reply docs/reference/deploy/master-config-reference.rst:243]: I don't see the helm/charts/determined/values.yaml part reverted? I consider that part of the public facing docs for this feature

**Review by lbliii — APPROVED** [review]

File: docs/reference/deploy/master-config-reference.rst (hunk 1 above)
Comment (tara-det-ai) [reply docs/reference/deploy/master-config-reference.rst:243]: that will have to be a code PR

**Pull request merged**